### DISCUSSION TITLE
2024-06-26-hexo-beautification

### DISCUSSION THREAD

**Discussion by giscus** [ouharu/ouharu.github.io#2] [Show and tell]
# 2024-06-26-hexo-beautification

参考： https://hexo.fluid-dev.com/docs/guide/ 1. fluid主题安装命令行输入： 12345678PS C:\Users> npm install --save hexo-theme-fluidadded 1 package, and audited 229 packages in 14s23 packages are looking for fun

http://localhost:4000/2024-06-26-hexo-beautification

**Comment by ouharu** [comment]
# comment test 
- 6/28
- abc
- 你好
- 👋
- こんにちは
